clients/python/llmengine/model.py

```diff
@@ -167,6 +167,94 @@ def create(
                 An optional dictionary of key/value pairs to associate with this endpoint
         Returns:
             CreateLLMEndpointResponse: creation task ID of the created Model. Currently not used.
+
+        === "Create Llama 2 7B model in Python"
+            ```python
+            from llmengine import Model
+
+            response = Model.create(
+                name="llama-2-7b-test"
+                model="llama-2-7b",
+                inference_framework_image_tag="2.0.0",
+                inference_framework=LLMInferenceFramework.VLLM,
+                num_shards=1,
+                checkpoint_path="s3://path/to/checkpoint",
+                cpus=8,
+                memory="24Gi",
+                storage="40Gi",
+                gpus=1,
+                min_workers=0,
+                max_workers=1,
+                per_worker=10,
+                endpoint_type=ModelEndpointType.STREAMING,
+                gpu_type="nvidia-ampere-a10",
+                public_inference=False,
+                labels={
```

Comment:
We might want to omit this, or put a separate `"product": "undefined"` value. We only want this particular `"llm_model_zoo"` label when we create the base model endpoints ourselves - we don't want other teams to use this.